packages/tooling/fast-tooling/src/web-components/html-render-layer-navigation/resize-observer.ts

```diff
@@ -0,0 +1,17 @@
+import type { ResizeObserverEntry } from "./resize-observer-entry";
+
+export type ConstructibleResizeObserver = new (
+    callback: ResizeObserverCallback
+) => ResizeObserverClassDefinition;
+
+export declare class ResizeObserverClassDefinition {
+    constructor(callback: ResizeObserverCallback);
+    public observe(target: Element): void;
+    public unobserve(target: Element): void;
+    public disconnect(): void;
+}
+
```

Comment:
All of these types and interfaces exist in ..anchored-region/resize-observer.ts, you should import those rather than duplicate the code. As an aside, we should probably move resize-observer somewhere more globally, like ../resize-observer/index.ts or into a definition file.

Reply:
I had advised duplicating since we are moving this code to a new repository in a few weeks, even though we could reference it it is located in anchored-region and we would have to specifically drill down into the package when pathing to reference. We will be moving this code and upgrading our TypeScript dependency then deleting these files so I think the duplication, due to it being temporary, is harmless for now.